src/pages/settings/InitialPage.js

```diff
@@ -8,19 +8,23 @@ import styles from '../../styles/styles';
 import Text from '../../components/Text';
 import {signOut} from '../../libs/actions/Session';
 import ONYXKEYS from '../../ONYXKEYS';
-import {version} from '../../../package.json';
 import AvatarWithIndicator from '../../components/AvatarWithIndicator';
 import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
 import Navigation from '../../libs/Navigation/Navigation';
 import {
-    Gear, Lock, Profile, Wallet, SignOut,
+    Gear,
+    Lock,
+    Profile,
+    Wallet,
+    SignOut,
+    Info,
 } from '../../components/Icon/Expensicons';
 import ScreenWrapper from '../../components/ScreenWrapper';
 import MenuItem from '../../components/MenuItem';
 import ROUTES from '../../ROUTES';
-import openURLInNewTab from '../../libs/openURLInNewTab';
-import CONST from '../../CONST';
-import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import withLocalize, {
```

Comment:
I don't think this is necessary.

Reply:
Why? If I remove this then I can't access `translate`

Reply:
Sorry, I meant the newline

Reply:
Done